app/src/main/res/values/strings.xml

```diff
@@ -222,6 +222,9 @@
     <string name="error_invalid_password">The password needs to be at least 6 characters long.</string>
     <string name="error_field_required">This field is required.</string>
     <string name="action_preferences">Preferences</string>
+     <string name="ok">OK</string>
+     <string name="cancel">CANCEL</string>
+    <string name="pressed_cancel">Log out cancelled</string>
     <string name="prefs_scan_startup">Scan on startup</string>
```

Comment:
Follow proper formatting if you want your travis build to pass.

Reply:
Can you merge this pull request with the one introducing the feature ?

Reply:
Yeah sure!!

Reply:
Thanks @samagra14  for the suggestions

Reply:
Yeah sure @teolemon